sos/report/plugins/ebpf.py

```diff
@@ -15,6 +15,7 @@ class Ebpf(Plugin, IndependentPlugin):
     short_desc = 'eBPF tool'
     plugin_name = 'ebpf'
     profiles = ('system', 'kernel', 'network')
+    commands = ('bpftool',)
```

Comment:
`commands` mean the plugin will be enabled any time some command from the tuple is present. So the line will be a new independent way how the plugin will be enabled by default.

If your aim is to enable the plugin automatically *only* when the binary is present, I think you need to test it in overwritten `check_enabled` method..?